local-dev/src/components/GlobalStatus.js

```diff
@@ -48,15 +45,51 @@ class GlobalStatus extends Component {
             default:
                 break;
         }
+        return img;
+    }
+    setFavicon(status) {
+        // update favicon
+        var favicon = document.getElementById('favicon');
+        var img = this.statusToIcon(status);
 
         if (img) {
             favicon.href = img;
         }
     }
+    realDesktopNotify(message, icon) {
+        var options = {
+            body: 'Mozilla Engagement Engineering Status Board',
+            icon: icon
+        }
+        new Notification(message, options);
+    }
+    desktopNotify(message, icon) {
+        if (!('Notification' in window)) {
+            return;
+        }
+        // Let's check whether notification permissions have already been granted
+        else if (Notification.permission === 'granted') {
+            this.realDesktopNotify(message, icon);
+        }
+        // Otherwise, we need to ask the user for permission
+        else if (Notification.permission !== 'denied') {
+            Notification.requestPermission(function (permission) {
+                // If the user accepts, let's create a notification
+                if (permission === 'granted') {
+                    this.realDesktopNotify(message, icon);
+                }
+            });
+        }
+    }
 
     render() {
         this.setFavicon(this.props.status);
 
+        if (this.props.status !== this.props.oldStatus &&
+            this.props.oldStatus && this.props.oldStatus !== 'pending') {
```

Comment:
I feel like it would read more clearly to do the `status !== oldStatus` check last (going from least complex check to most complex check), but I could just be picking nits. 😁